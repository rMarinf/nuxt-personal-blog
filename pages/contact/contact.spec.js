import { shallowMount } from '@vue/test-utils'
import Contact from './index.vue'

test('Contact', () => {
  // render the component
  const wrapper = shallowMount(Contact)

  expect(wrapper.element).toMatchSnapshot()
})
