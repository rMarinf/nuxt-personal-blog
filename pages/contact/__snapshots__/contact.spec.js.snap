// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact 1`] = `
<div
  id="contact"
>
  
  CONTACT

</div>
`;
